octree_web_viewer/client/package.json

```diff
@@ -29,4 +29,4 @@
     "single-quote": true,
     "trailing-comma": "es5"
   }
-}
+}
```

Comment:
Please add a newline here.
You can also add `"files.insertFinalNewline": true` to your VS Code's `settings.json`, so this will never happen again.

Reply:
indeed, i finded it eventually in the settings!